Cargo.lock

```diff
@@ -733,7 +733,7 @@ dependencies = [
  "bitflags 2.6.0",
  "cexpr",
  "clang-sys",
- "itertools 0.13.0",
+ "itertools 0.11.0",
```

Comment:
just don't get cargo update doing this mess, downgrading itertools and upgrading windows-targets

Reply:
Yeah, at this point I ignore this mess, as long as it doesn't introduce new duplication (still have that for those versions which is unfortunate)

Reply:
ok, i updated a couple spots. i'll land it when it's green